Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/TeleOpCompetition.java

```diff
@@ -1,7 +1,6 @@
 package org.firstinspires.ftc.team417_PowerPlay;
```

Comment:
Please don't merge this file.  You're not allowed to change another team's code.

Reply:
Not sure how this happened, but it will be changed

Reply:
This has been resolved by the latest push